README.rst

```diff
@@ -15,21 +15,13 @@ A basic call to SQS Workers would look like this:
 
 .. code:: python
 
-    from sqsworkers import MessageMetadata
     from sqsworkers.crew import Crew
 
 
-    class MyMessageProcessor:
-        def __init__(self, msg):
-            self.metadata = MessageMetadata(msg)
-
-        def start(self):
-            print(f"metadata: {self.metadata}")
-
     crew = Crew(
         sqs_session=my_session,
         queue_name=my_queue_name,
-        message_processor=MyMessageProcessor
+        message_processor=lambda msg: f"hello, {msg}"
```

Comment:
is this right? What does this do?

Reply:
In this case, nothing. It would just return the string. It's just to show that message_processor can be any callable -- it doesn't have to be a full-fledged class with a `.start` method

Reply:
I updated the readme to make it a bit clearer, since the whole point of using sqsworkers is to have some side-effect occur given a message from an sqs queue. The readme now has the function print the message to the screen